science/gds-dtt/Portfile

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem    1.0
+PortGroup     conflicts_build 1.0
+PortGroup     active_variants 1.1
+PortGroup     python 1.0
+
+name          gds-dtt
+version       2.19.0
+categories    science
+platforms     darwin
+maintainers   {@emaros ligo.org:ed.maros} openmaintainer
+license       GPL
+
+description   LSC Algorithm Library
+long_description \
+  LIGO Scientific Collaboration Algorithm Library containing core \
+  routines for gravitational wave data analysis.
+
+homepage      https://www.lsc-group.phys.uwm.edu/daswg/projects/lalsuite.html
+master_sites  http://software.ligo.org/lscsoft/source/
+
+checksums     rmd160 45f5f06d8190038ba43dede6a6d409467214687d \
+              sha256 8eb3c03845f5fa00bf9b4da104c40cb7773868570ac61c926befffa515f54f60
+
+depends_build-append \
+              port:pkgconfig \
+              port:swig \
+              port:swig-python \
+              port:gds \
+              port:gds-gui \
+              port:gds-swig
+
+depends_lib   port:expat \
+              port:fftw-3 \
+              port:fftw-3-single \
+              port:jsoncpp \
+              port:ldas-tools-framecpp \
+              port:metaio
+
+#------------------------------------------------------------------------
+# Fix things after PortGroup python
+#------------------------------------------------------------------------
+use_configure      yes
+use_parallel_build yes
+build.cmd          make
+build.target       all
+destroot.cmd       ${build.cmd}
+destroot.target    install
+destroot.destdir   DESTDIR=${destroot}
+destroot.args      " "
+destroot.pre_args      ${destroot.target}
+destroot.post_args      ${destroot.destdir}
+
+configure.python ${prefix}/bin/python2.7
+configure.args --enable-dtt \
+               --enable-python \
+               --enable-root-objects \
+               --without-sasl \
+               --with-jsoncpp=${prefix}
+
+#------------------------------------------------------------------------
+# Use root 6 for High Sierra and beyond
+# else root 5
+#------------------------------------------------------------------------
+if {${os.platform} eq "darwin" && ${os.major} >= 17 } {
+    use_autoreconf yes
+    use_automake yes
+
+    set python.version      36
+    destroot.args-append \
+        pythondir="${python.pkgd}" \
+        pyexecdir="${python.pkgd}" \
+        pkgpythondir="${python.pkgd}/${name}" \
+        pkgpyexecdir="${python.pkgd}/${name}"
+
+    configure.env           CXXFLAGS='-std=c++1z'
+
+    depends_lib-append      port:root6 \
+                            port:python${python.version} \
+                            port:py${python.version}-numpy
+
+	require_active_variants root6 python${python.version}
+} else {
+    set python.version      27
+    configure.env           CXXFLAGS='-std=c++11'
```

Comment:
when adding this flag you likely want to make sure to request a C++11 compatible compiler, so with `compiler.cxx_standard   2011` perhaps?